docs/UserGuide.md

```diff
@@ -189,7 +192,9 @@ Sample Output for `stats PGPH`:
 
 ### Adding an appointment: `addAppt`
 
-Adds an appointment to Vax@NUS' records. 
+Adds an appointment to Vax@NUS' records.
+
+The current version of Vax@NUS allows no more than one appointment scheduled for each student. You thus won't be able to add a new appointment for a student if they already have one.
```

Comment:
I think it's better to say 'will not' instead of 'won't'

Reply:
Maybe can phrase so its simpler English like "Therefore, you will not be able to add a new appointment for a student if they already have one."